src/controllers/factorycontroller.js

```diff
@@ -105,6 +119,22 @@ class FactoryController extends OrderController
         return true;
     }
 
+    extraTruck (formValues)
+    {
+        if (this.validateTruck(truck)) {
+            GAME.model.config.money - GAME.model.config.costExtraTruck);
```

Comment:
You'll need to assign this, or use `this.updateMoney(-GAME.model.config.costExtraTruck)`